content/fast/use-imagemin-to-compress-images/codelab-imagemin-webpack.md

```diff
@@ -77,9 +77,9 @@ You now have a Webpack config that compresses images using Imagemin.
 
 ## Run Webpack
 
-- Click the **Tools** button.
-- Then click the **Console** button.
-- To compress your images, run Webpack by typing the following command into the 
+1. Click the **Tools** button.
+2. Then click the **Console** button.
```

Comment:
You can do `1.` for all of these and it will generate the right counters. I think it's easier to stick with `1.` because you don't have to renumber the steps if you remove one from the middle.